src/fides/api/ctl/routes/system.py

```diff
@@ -0,0 +1,211 @@
+from typing import List
+
+from fastapi import Depends, HTTPException, Security
+from fastapi_pagination import Page, Params
+from fastapi_pagination.bases import AbstractPage
+from fastapi_pagination.ext.sqlalchemy import paginate
+from loguru import logger
+from pydantic.types import conlist
+from sqlalchemy.orm import Session
+from starlette.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_404_NOT_FOUND,
+    HTTP_422_UNPROCESSABLE_ENTITY,
+)
+
+from fides.api.ctl.sql_models import System  # type: ignore[attr-defined]
```

Comment:
Mypy was having a false positive saying that `System` didn't exist in that module.